app/models/post.rb

```diff
@@ -2,4 +2,12 @@ class Post < ApplicationRecord
   belongs_to :user
   has_many :likes
   has_many :comments
+
+  def update_posts_counter
+    Post.find_by(:id, post_id).count.update
+  end
```

Comment:
- [ ] I think, in this case, you have to find the post which is written by the specific user. So, line 7 would be 

```ruby
Post.find_by(:id, user_id).count.update

 #or you can write in another way as well

user.increment!(:posts_counter)
```

Reply:
Finally:smile: address the same issue here, please.

Reply:
This issue is well resolved! ✔️